### PR TITLE
update config in db

### LINKED ISSUE #66: make rich precense better structure in db + use in code

### LINKED ISSUE #75: Move hardcoded tmwiki.be to DB

### PR DESCRIPTION
**Pull request by PandaGamersHDNL**

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -356,20 +356,20 @@ export const commands: Command[] = [
     aliases: [],
     async response(msg: Message, guildConfig: GuildConfig | undefined): Promise<Response | void> {
       const tokenizer = new Tokenizer(msg.content, guildConfig?.prefix || defaultPrefix);
-
+      const link = (await ConfigService.get()).wiki;
```
Comment (DrunkenToast) [inline src/commands.ts]: URL prefered but doestn't matter

File: src/discord.ts [hunk 2]
```diff
@@ -123,23 +90,24 @@ export async function buildClient(shard: number, shardCount: number): Promise<Cl
       }
       return;
     }
-    const commandList = commands.concat(guildConfig.modules['customCommands'] === false ? [] : guildConfig.commands);
+    const commandList = commands.concat(guildConfig === undefined || guildConfig.modules['customCommands'] === false ? [] : guildConfig?.commands);
```
Comment (DrunkenToast) [inline src/discord.ts]: == falsey checks

File: src/discord.ts [hunk 3]
```diff
@@ -123,23 +90,24 @@ export async function buildClient(shard: number, shardCount: number): Promise<Cl
...
+
     for (const command of commandList) {
       if (tokenizer.command() !== command.name && !command.aliases.includes(tokenizer.command()!)) {
         continue;
       }
-      if (guildConfig.modules[command.category] === false) {
-        msg.channel.send(new EmbedBuilder().error(`the ${command.name} command has been disabled` , `enable the ${command.category} module to enable this command`));
+      if (guildConfig && guildConfig.modules[command.category] === false) {
```
Comment (DrunkenToast) [inline src/discord.ts]: == falsey

File: src/models/config.ts [hunk 4]
```diff
@@ -1,19 +1,12 @@
...
-      states: [
-        {
-          status: PresenceStatusData | string,
-          activity: {
-            name: string,
-            type: number
-          }
-        }
-      ]
+      states: PresenceData[]
```
Comment (DrunkenToast) [inline src/models/config.ts:8]: consider removing states and just an array

**Review by DrunkenToast — CHANGES_REQUESTED** [review]

File: src/discord.ts (hunk 2 above)
Comment (PandaGamersHDNL) [reply src/discord.ts]: no only not include when false

File: src/models/config.ts (hunk 4 above)
Comment (PandaGamersHDNL) [reply src/models/config.ts:8]: interval is included in presence

**Review by DrunkenToast — APPROVED** [review]

**Review by xunafay — APPROVED** [review]

**Pull request merged**